engine/runtime/audit_delegate.py

```
@@ -379,32 +379,28 @@ async def run_chapter_audit(host: Any, novel: Novel) -> None:
+            (novel.novel_id.value, chapter_num), None
+        )
+        voice_result = drift_result if drift_result.get("similarity_score") is not None else None
+        drift_result = await host._call_with_timeout(
+            host.aftermath_pipeline.run_after_chapter_saved(
+                novel.novel_id.value,
+                chapter_num,
+                content,
+                chapter_micro_beats=_mb,
+                voice_result=voice_result,
+            ),
+            timeout=300.0,  # 章后管线最多 5 分钟（含多次 LLM）
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)